core/backend/package.json

```
@@ -22,11 +22,10 @@
     "lint": "eslint -f visualstudio \"./src/**/*.ts\" 1>&2",
     "pretest": "cpx ./src/test/logging.config.json ./lib/test",
     "test": "betools test --offline=\"mock\" --grep=\"#integration|#WebGLPerformance\" --invert",
-    "runIntegrationTests": "betools test --testDir=\"./lib/test/integration\"",
-    "setupAzurite": "mkdir .\\azuriteStorage && azurite --silent --location ./azuriteStorage",
-    "cleanupAzurite": "rimraf ./azuriteStorage",
-    "runIntegrationTestsWithAzurite": "run-p -r setupAzurite runIntegrationTests",
-    "test:integration": "run-s -c pretest runIntegrationTestsWithAzurite cleanupAzurite",
+    "azurite:setup": "mkdir ./lib/test/azuriteStorage && azurite --silent --location ./lib/test/azuriteStorage",
+    "test:integration": "run-s pretest test:integration:azurite",
+    "test:integration:azurite": "run-p azurite:setup test:integration:core",
```

Needs -r or --race so that run-p terminates all tasks when test:integration:core exits, otherwise it will hang while the azurite server continues running after the tests finish.
```suggestion
    "test:integration:azurite": "run-p -r azurite:setup test:integration:core",
```

---

I was hitting the opposite for some reason though.  The azurite was closing prior to the integration tests being done... Maybe a timeout scenario?

I'll dig more into the case I was seeing.

---

@skirby1996 looks like the integration tests were having issues and one of them was crashing.  I talked to @diegopinate about it and he's going to take a look. For now, I'll just push a change to skip that test.